stores/nvidia.py

```diff
@@ -52,7 +52,7 @@
 NVIDIA_TOKEN_URL = "https://store.nvidia.com/store/nvidia/SessionToken"
 
 GPU_DISPLAY_NAMES = {
-    "2060S": "NVIDIA GEFORCE RTX 2060 SUPER",
+    "Black_t-shirt": "NVIDIA Logo Graphic Tee - Black", # availableQuantity	2147483647
```

Comment:
Can you make this an addition instead of a replace?
```
GPU_DISPLAY_NAMES = {
    "Black_t-shirt": "NVIDIA Logo Graphic Tee - Black",
    "2060S": "NVIDIA GEFORCE RTX 2060 SUPER",
    "3080": "NVIDIA GEFORCE RTX 3080",
    "3090": "NVIDIA GEFORCE RTX 3090",
}
```